api/lib/application/certification-courses/index.js

```diff
@@ -5,7 +5,31 @@ import { certificationCourseController } from './certification-course-controller
 import { identifiersType } from '../../../src/shared/domain/types/identifiers-type.js';
 
 const register = async function (server) {
-  server.route([
+  const adminRoutes = [
+    {
+      method: 'PATCH',
+      path: '/api/admin/certification-courses/{id}',
+      config: {
+        validate: {
+          params: Joi.object({
+            id: identifiersType.certificationCourseId,
+          }),
+        },
+        handler: certificationCourseController.update,
+        tags: ['api'],
```

Comment:
Un tag que l'on retrouve sur d'autres routes

```suggestion
        tags: ['api', 'admin'],
```